### PR TITLE
Adds bikehorn "suppressor"

### PR DESCRIPTION
**Pull request by Moltijoe**
https://forums.yogstation.net/threads/7-bikehorn-suppressors-for-the-stechkin-ss13-server.25595/

making them attainable wasn't part of the bounty's scope, someone else can do that, or hisa can just spawn them in

the item
item

when attached
attached

:cl:  
rscadd: Adds bikehorn suppressor
imageadd: adds icon for the bikehorn suppressor item and an overlay version for all existing suppressor variants
/:cl:

### CODE REVIEW

**Comment by cuackles** [comment]
Sprite has a lot of banding and no outlines, i would recommend reducing the pallet to like 3 maybe 4 shades of reds, using the darker for the outline and using, you can probably just edit the actually honk horn using parts of the silencer to make it look attachable, the attached icon also seems a bit off, but i do not remember how the clasic silencer looks on the new sketching sprite so i would have to see that tomorrow to be able to give consice help

**Comment by cuackles** [comment]
Actually maybe just flip the attached sprite upside down and add a slight outline and should be good, cus you can see that the sketching is illuminated at the top but the attached item is illuminated on the bottom and that breaks the something something look of it

**Comment by Moltijoe** [comment]
> Sprite has a lot of banding and no outlines, i would recommend reducing the pallet to like 3 maybe 4 shades of reds, using the darker for the outline and using, you can probably just edit the actually honk horn using parts of the silencer to make it look attachable, the attached icon also seems a bit off, but i do not remember how the clasic silencer looks on the new sketching sprite so i would have to see that tomorrow to be able to give consice help

clown horn has a one direction shading
suppressor has, pillow shading was it?
image

**Comment by Moltijoe** [comment]
swapped it to pillow shading and reduced the number of reds used, how's this?
image

**Comment by Moltijoe** [comment]
new in-game images

image

image

**Comment by cuackles** [comment]
> swapped it to pillow shading and reduced the number of reds used, how's this?
> image

Ok yeah i will probably never be able to unsee the banding and shading of the silencer now, ok so the new one, it does look better yes, but you should not be using pillow shading as, hhhhhhhhmmmmm i don't really know how to put it but it's not good practice, try to imagine it has a light source and according to that place lighter points near the light and darker where the imaginary light woundnt reach as easily, preferably make it follow the shading the sketching has to it's kept consistent while attached

**Comment by cuackles** [comment]
> new in-game images
> 
> image
> 
> image
> 

Overlay looks better, me sleep now continue helping tomorrow but it's really close to where it has to be to be to be to be accepted

**Comment by Moltijoe** [comment]
new has a border, but also shading

image

**Comment by cuackles** [comment]
Lots lots better, give me a few hours to finish all my human responsibilities and i will take a good look at it to see if there is anything that can be improved

**Comment by cuackles** [comment]
ok after seeing the current icon files, I think the item is good and can be accepted but i would like to see if you are able to give the same shading and color correcting treatment that you gave to the item to the attachment sprite, mostly a noticeable outline and make the lips of the attachments be similar to each other, mainly the pistol one that looks like it was mushed and the lips go a bit to outwards

**Comment by Moltijoe** [comment]
> ok after seeing the current icon files, I think the item is good and can be accepted but i would like to see if you are able to give the same shading and color correcting treatment that you gave to the item to the attachment sprite, mostly a noticeable outline and make the lips of the attachments be similar to each other, mainly the pistol one that looks like it was mushed and the lips go a bit to outwards

there's very little room to work with for the pistol one

**Comment by Moltijoe** [comment]
image
fixed it up, but it still seems to connect weirdly, gonna make it 1 pixel shorter and shift it up by 1

**Comment by Moltijoe** [comment]
image
i think this looks nicer

**Review by cuackles — APPROVED** [review]
honk

File: code/modules/projectiles/guns/ballistic.dm [hunk 1]
```diff
@@ -616,9 +648,16 @@ GLOBAL_LIST_INIT(gun_saw_types, typecacheof(list(
...
-
 /obj/item/suppressor/specialoffer
 	name = "cheap suppressor"
 	desc = "A foreign knock-off suppressor, it feels flimsy, cheap, and brittle. Still fits most weapons."
 	icon = 'icons/obj/guns/projectile.dmi'
 	icon_state = "suppressor"
+
+/obj/item/enloudener
+	name = "bikehorn \"suppressor\""
+	desc = "Advanced clown research has found that guns that honk shoot harder, faster and more accurately. (they don't)"
```
Comment (ReddicusDragon) [inline code/modules/projectiles/guns/ballistic.dm]: Please capitalize the T in "They"

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

**Review by ReddicusDragon — APPROVED** [review]
spelling wise, looks good. Have not checked the actual code in this PR itself.

**Comment by JamieD1** [comment]
god help us

**Pull request merged**